ambry-frontend/src/main/java/com/github/ambry/frontend/AmbrySecurityService.java

```diff
@@ -235,6 +246,10 @@ public void processResponse(RestRequest restRequest, RestResponseChannel respons
         exception = e;
       }
     }
+    if (quotaConfig.requestQuotaThrottlingEnabled) {
+      // TODO use enforcement recommendation to create more specific error message and set specific response headers.
+      quotaManager.charge(restRequest, blobInfo, new ArrayList<>());
+    }
```

Comment:
If this is only a TODO, I recommend now to call charge at all. just leave a comment here.